src/components/Inventory/NurseryDropdown.tsx

```diff
@@ -35,7 +35,7 @@ export default function NurseryDropdown<T extends AccessionPostRequestBody>(
       selectedValue={record.facilityId?.toString()}
       options={getAllNurseries(organization).map((nursery) => ({ label: nursery.name, value: nursery.id.toString() }))}
       onChange={onChangeHandler}
-      errorText={validate && !record.facilityId ? strings.REQUIRED_FIELD : ''}
+      errorText={validate && (!record.facilityId || record.facilityId === -1) ? strings.REQUIRED_FIELD : ''}
```

Comment:
under what condition is the `record.facilityId` equal to -1?

Reply:
gripe,, unfortunately it is initiallized as -1 in calling code since facilityId cannot be null/undefined
cc @constanzauanini

Reply:
Refactoring this to avoid assumptions